package.json

```diff
@@ -13,7 +13,11 @@
   },
   "homepage": "https://github.com/Seneca-CDOT/telescope#readme",
   "dependencies": {
+    "addressparser": "^1.0.1",
     "bent": "^7.0.2",
-    "bull": "^3.11.0"
+    "bull": "^3.11.0",
+    "feedparser": "^2.2.9",
+    "mri": "^1.1.4",
+    "readable-stream": "^3.4.0"
```

Comment:
This also seems unused.  Make sure you don't add dependencies that you don't use.